types/v56/types.go

```diff
@@ -2171,6 +2194,48 @@ type ExternalNetworkReference struct {
 	Name string `xml:"name,attr,omitempty"`
 }
 
+// Type: VimObjectRefType
+// Namespace: http://www.vmware.com/vcloud/extension/v1.5
+// https://vdc-repo.vmware.com/vmwb-repository/dcr-public/7a028e78-bd37-4a6a-8298-9c26c7eeb9aa/09142237-dd46-4dee-8326-e07212fb63a8/doc/doc/types/VimObjectRefsType.html
+// Description: Represents the moref and the type of a vSphere object.
```

Comment:
"moref" ?

Reply:
Yep - `moref` - this is vCD API - that's how it's called

Reply:
Can we say "Managed Object Reference (MoRef)" for clarity?

Reply:
Done